book/src/user/performance.md

```diff
@@ -0,0 +1,53 @@
+## Improving Performance
```

Comment:
Some of these tips seem like troubleshooting tips. I wonder if we want to merge this section and the "Troubleshooting" section or maybe link to it from Troubleshooting.

Reply:
I was wavering over the same thing as well. I'll merge the contents of the two sections, remove "Improving Performance" and keep only "Troubleshooting".

My rationale for having those two was that in the future we might be adding more content there, and having it split would prevent the section from being too long.

Reply:
Done in [85a41e2](https://github.com/ZcashFoundation/zebra/pull/6733/commits/85a41e27291c4cc7001a0b8f7706a1b42365075e).